packages/trader/build/constants.js

```diff
@@ -120,7 +120,13 @@ const plugins = (base, is_test_env, is_mocha_only) => ([
     ...(is_test_env && !is_mocha_only ? [
         new StylelintPlugin(stylelintConfig()),
     ] : [
-        new SWPrecacheWebpackPlugin(swPrecacheConfig(base)),
+        new GenerateSW({
+            importWorkboxFrom: 'local',
+            cleanupOutdatedCaches: true,
+            exclude: [/CNAME$/, /index\.html$/, /404\.html$/],
+            skipWaiting: true,
+            clientsClaim: true,
+        })
```

Comment:
Can we move the config here to the `build/config.js` file to keep consistency?

Reply:
Yes sir! right away.